tests/unit/ed25519.test.ts

```diff
@@ -49,6 +49,16 @@ describe("Ed25519PublicKey", () => {
     ).toBe(false);
   });
 
+  it("should fail malleable signatures", () => {
+    const signature = new Ed25519Signature(
+      // eslint-disable-next-line max-len
+      "ad0828843228048fee9ead35cf0decc8155342e967993851f70bfc0bb9e97719a094c6f956e5cd65872fa769e7dce26f6e637eb4ff70c3f6aafc5f06fbc300ad",
```

Comment:
Can you document how you obtained this?

Reply:
This is from the GitHub issue... I've been unable to generate a non-canonical signature

Reply:
Cool! Just add a link to the GH issue.

To obtain a non canonical signature, just take `L` and increment it. Then set `s` to be that value. :) Then, set the other component of the signature to be junk, since verification will short-circuit on `s`.

Reply:
Alternatively, create a normal canonical signature, take its `s` and set it to `s + L` or `s + 2L` or any `s + kL` for `k > 0`.

Reply:
Fixed to test L and L + 1